rule-example.yaml

```diff
@@ -29,7 +29,7 @@
     or:
     - builtin.xml:
         xpath: "//dependencies/dependency"
-        filepaths: "{{poms.filepaths}}"
+        filepaths: "{{poms.extras.filepaths}}"
```

Comment:
Updating to extras to make sure that the TemplateContext works, we should maybe have another test that tests the file paths from incidents are working.